Update Linux perfcollect docs to disable Write-No-Execute (#32338)
dotnet/docs: docs/core/diagnostics/trace-perfcollect-lttng.md

    > export DOTNET_EnableEventLog=1
    > ```

    > [!NOTE]

>     > When executing the app with .NET 7, please set `DOTNET_EnableWriteXorExecute=0` in addition to the above environment variables.  For example:

Reviewer: ```suggestion
    > When executing the app with .NET 7, you must  also set `DOTNET_EnableWriteXorExecute=0` in addition to the preceding environment variables.  For example:
```

> 

Reviewer: ```suggestion
>
```

> [!NOTE]
> Events written via <xref:System.Diagnostics.Tracing.EventSource?displayProperty=nameWithType> API (including the events from Framework) won't show up under their provider name. Instead, they are written as `EventSourceEvent` events under `Microsoft-Windows-DotNETRuntime` provider and their payloads are JSON serialized.

> [!NOTE]

> > If you observe `[unknown] /memfd:doublemapper` frames in method names and callstacks, please set `DOTNET_EnableWriteXorExecute=0` before running the app being traced with perfcollect.

Reviewer: ```suggestion
> If you observe `[unknown] /memfd:doublemapper` frames in method names and callstacks, set `DOTNET_EnableWriteXorExecute=0` before running the app that you're tracing with perfcollect.
```
